includes/privacy.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+namespace WordCamp\Budgets\Privacy;
+
+use WP_Query;
+use WordCamp\Budgets\Reimbursement_Requests;
+
+defined( 'WPINC' ) || die();
+
+
+add_filter( 'wp_privacy_personal_data_exporters', __NAMESPACE__ . '\register_personal_data_exporters' );
+add_filter( 'wp_privacy_personal_data_erasers', __NAMESPACE__ . '\register_personal_data_erasers' );
+
+/**
+ * Registers the personal data eraser for each WordCamp post type
+ * @param array $erasers
+ *
+ * @return array
+ */
+function register_personal_data_erasers( $erasers ) {
+	/**
+	 * We should not add a eraser for WordCamp post type, because it contains data which can be used for
+	 * accounting or reference purpose.
+	 *
+	 * This is just an empty stub so that we do not create one.
+	 */
+	return $erasers;
+}
+
+/**
+ * Registers the personal data exporter for each WordCamp post type.
+ *
+ * @param array $exporters
+ *
+ * @return array
+ */
+function register_personal_data_exporters( $exporters ) {
+	$exporters['wcb-reimbursements'] = array(
+		'exporter_friendly_name' => __( 'WordCamp Reimbursement Requests', 'wordcamporg' ),
+		'callback'               => __NAMESPACE__ . '\reimbursements_exporter',
+	);
+
+	$exporters['wcb-vendor-payments'] = array(
+		'exporter_friendly_name' => __( 'WordCamp Vendor Payment Requests', 'wordcamporg' ),
+		'callback' => __NAMESPACE__ . '\vendor_payment_exporter',
+	);
+
+	return $exporters;
+}
+
+/**
+ * Finds and exports personal data associated with an email address in a vendor payment request
+ *
+ * @param string $email_address
+ * @param int $page
+ *
+ * @return array
+ */
+function vendor_payment_exporter( $email_address, $page) {
+
+	$results = array(
+		'data' => array(),
+		'done' => true,
+	);
+
+	$user = get_user_by( 'email', $email_address );
+
+	if ( empty( $user->ID ) ) {
+		return $results;
+	}
+
+	$sponsor_invoices = wcb_get_post_wp_query( \WCP_Payment_Request::POST_TYPE, $page, $user->ID );
+
+	if ( empty( $sponsor_invoices ) ) {
+		return $results;
+	}
+
+	$data_to_export = array();
+	foreach ( $sponsor_invoices->posts as $post ) {
+		$sponsor_inv_exp_data = array();
+		$meta = get_post_meta( $post->ID );
+
+		$sponsor_inv_exp_data[] = [
+			'name' => 'Title',
+			'value' => $post->post_title,
+		];
+		$sponsor_inv_exp_data[] = [
+			'name' => 'Date',
+			'value' => $post->post_date,
+		];
+
+		$sponsor_inv_exp_data =
```

Comment:
This is personal preference, but I find it harder to read when the line ends at the equal sign. I'd probably do it like this instead (but up to you):

```
		$sponsor_inv_exp_data = array_merge(
			$sponsor_inv_exp_data,
			wcb_get_meta_details( $meta, '_camppayments_' )
		);
```